D/DynarePreprocessor/build_tarballs.jl

```diff
@@ -21,9 +21,14 @@ rm -f /usr/bin/flex
 
 # Help FreeBSD find header files.  See
 # https://github.com/JuliaPackaging/Yggdrasil/issues/3949
+if [[ "${target}" == *-freebsd* ]] || [[ "${target}" == *-apple-* ]]; then
+    CC=gcc
+    CXX=g++
+    LDFLAGS=-static-libgcc
+fi
```

Comment:
We generally don't want to use GCC on BSD systems.  Why would be that necessary?